Automattic/wp-calypso
# Pro Dashboard: Refactor Notification setting contact form.

This PR unifies the SMS and Email editor implementation by adding a shared component that handles both contact type.

Related to 1204774821045518-as-1204802449809227

## Proposed Changes

* Add a Contact Editor component that unifies the implementation for both SMS and Email contact form.
* Split the editor to separate to improve the manageability of the codes.

## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

* Run git checkout add/downtime-monitoring-contact-list-common-component and yarn start-jetpack-cloud
Open http://jetpack.cloud.localhost:3000/, and you'll be redirected to the /dashboard.
Enable monitor if not enabled already.
Click on the clock icon next to the monitor toggle.
Confirm that the email and SMS editor work as expected.

## Pre-merge Checklist
